src/pages/HomePage.js

```diff
@@ -14,11 +14,13 @@ export default function HomePage({ cards, catches }) {
       <main>
         <Title>Tight lines, Caspar</Title>
         <CardsList>
-          {cards.map((data, index) => (
-            <li key={index}>
-              <Cards data={data} catches={catches} />
-            </li>
-          ))}
+          {cards
+            ? cards.map((data, index) => (
+                <li key={index}>
+                  <Cards data={data} catches={catches} />
+                </li>
+              ))
```

Comment:
You do have an `id` in your data, right? If so, then it's always preferable to use the `id` as key instead of `index`.
```suggestion
            ? cards.map((data) => (
                <li key={data.id}>
                  <Cards data={data} catches={catches} />
                </li>
              ))
```

Reply:
I have an id! I'll change that